src/tracing/esp32_trace/include/matter/tracing/macros_impl.cpp

```diff
@@ -17,8 +17,135 @@
 #include "macros_impl.h"
 #include <esp_heap_caps.h>
 #include <esp_insights.h>
+
 namespace Insights {
 
+/* The traces of PacketParser,MinMdnsResolver and SesssionManager are filtered out :
+ * i) To report the traces of more concern during commissioning like CASESession,PASESession,
+ * traces for OperationalSessionSetup, GeneralCommissioning and NetworkCommissioning
+ * ii) although providing the user with ablity to add or remove the strings whose traces
+ * can be filtered out.
+ */
+namespace {
+static constexpr size_t kBlackListSize = 10;
+static bool isBlacklistInitialized     = false;
```

Comment:
consider naming these with prefixes, like `gIsBlackListInitialized` and `gBlackListHashes` and `gBlackListStrings`